trustmanager/keyfilestore.go

```diff
@@ -301,10 +302,14 @@ func removeKey(s LimitedFileStore, cachedKeys map[string]*cachedKey, name string
 
 // Assumes 2 subdirectories, 1 containing root keys and 1 containing tuf keys
 func getSubdir(alias string) string {
-	if alias == "root" {
+	switch {
+	case alias == data.CanonicalRootRole:
 		return rootKeysSubdir
+	case data.ValidRole(alias):
```

Comment:
:|  I've changed this to BaseRole as suggested, so we put all delegations in there.  The tests seem to pass @endophage do you remember what was failing